Configs/.config/hypr/keybindings.conf

```diff
@@ -24,14 +24,14 @@ bind = Alt, Return, fullscreen, # toggle the window between focus and fullscreen
 bind = $mainMod, L, exec, swaylock # launch lock screen
 bind = $mainMod+Shift, F, exec, $scrPath/windowpin.sh # toggle pin on focused window
 bind = $mainMod, Backspace, exec, $scrPath/logoutlaunch.sh # launch logout menu
-bind = Ctrl, Escape, exec, killall waybar || waybar # toggle waybar
+bind = Ctrl+Alt,W , exec, killall waybar || waybar # toggle waybar
```

Comment:
I know it's very small, but can you add a space after the comma for the keybind to keep it matching the rest of the config?

Reply:
ok